R/trackplot.R

```diff
@@ -2397,7 +2427,7 @@ summarize_homer_annots = function(anno, sample_names = NULL, legend_font_size =
 }
 
 .check_mysql = function(warn = FALSE){
-  check = as.character(Sys.which(names = 'mysql'))[1]
+  check = as.character(Sys.which(names = 'mariadb'))[1]
```

Comment:
I am using MariaDB on Arch Linux, and the mysql package is being deprecated. It is expected to be removed in a future release, and it's suggested to use mariadb instead. By the way, MariaDB is a fork of MySQL , which is originated from the same source but has better performents and easy to install.

Reply:
I see. Thank you. Just FYI, there is a high possibility that the code base will be changed to use the [RMySQL](https://github.com/r-dbi/RMySQL) or the [RMariaDB](https://github.com/r-dbi/RMariaDB) R packages instead of command line SQL queries.

Reply:
Aye, by the way, I've retained the original `mysql` command in a separate commit.

Reply:
Yup. I will review them soon.